scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -1429,6 +1446,70 @@ while true; do
     fi
   fi
 
+      # Mithril Signer Section
+      mithrilSignerVars
+
+      if [[ "$nodemode" = "Core" ]] && [[ "$serviceExists" -eq 0 ]] && [[ "$signerServiceStatus" == "running" ]] && [[ "$metricsEnabled" == "true" ]]; then
```

Comment:
This line prevents gLiveview from showing the Mithril Signer is offline since it requires the state to be running.

Reply:
An SPO may decide to copy the required KES hot secret key onto a relay to run a signer decoupled from their Core/Block Producer. They also may have a misconfigured, or previously setup environment missing the variable to enable the metrics server. Since there are multiple variations of how/why the SPO may adjust things compared to our expectations I'm changing this to simply verify that **MITHRIL_SIGNER_ENABLED** is Y. Then the SPO will know if its online, or if offline that something needs to be further investigated.

```suggestion
      if [[ "${MITHRIL_SIGNER_ENABLED}" == "Y" ]] ; then
```